### ISSUE TITLE
Wiki.wemos.cc is offline

### ISSUE THREAD

**Issue by FloStar3000** [wemos/docs#6]
Hi, the wiki.wemos.cc site is offline! Can you please fix it? We need the documentation for the deprecated LOLIN32 and LOLIN32 Lite boards.
Have a nice day!
Florian

**Comment by akostadinov** [comment]
Yes, please. Being obsolete doesn't mean documentation should be removed. Can anybody upload the docs for it here at least?
